checklists/newHire.json

```diff
@@ -189,7 +190,7 @@
 	},
 	"signIntoEmployeeExpress": {
 		"displayName": "Sign into Employee Express",
-		"description": "This is available after your first paycheck. <a href=\"https://handbook.18f.gov/gsa-tools-equipment-and-transit/#employee-express\">Instructions</a>",
+		"description": "This is available after your first paycheck. You will receive an invite email from GSA HR.<a href=\"https://handbook.18f.gov/gsa-tools-equipment-and-transit/#employee-express\">Instructions</a>",
```

Comment:
Please add a space after `HR.` so that the link to `Instructions` doesn't abut the period.